server/checker/github.py

```diff
@@ -38,8 +40,23 @@ def get_readme_len(self):
         readme = urlopen(
             'https://raw.githubusercontent.com/%s/%s/master/README.md' % (self.owner, self.repo)
         ).read().strip().decode('utf-8')
+        result = []
+        if len(readme) > 0:
+            #readme exists
+            #any topics which should be addressed in a 
+            #readme will be checked by word movers distance
```

Comment:
`github.py` should only provide stuff from github, all checks should be implemented as a separate criterion in `criteria.py`

In that case we should rename  `get_readme_len` as `get_readme_content` and return the all content of `README`.
And move below code, and  `word_movers` to a new criterion.